source/nanoFramework.Json/JsonConvert.cs

```diff
@@ -520,7 +587,7 @@ private static JsonToken ParseValue(ref LexToken token)
 			}
 
 			throw new Exception("invalid value found during json parse");
-		}
+		}   // end of ParseValue()
```

Comment:
No need for comments like this, as VS can easily shrink functions and this does not conform to the C# nF code style

Reply:
OK